configuration/Makefile

```diff
@@ -0,0 +1,8 @@
+stop:
+	docker-compose down -v
```

Comment:
there are no named volumes, so `-v` is unnecessary?
(data won't be removed by `make stop` as this doesn't touch the host folders mounted as volumes)